activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Fix query attribute method on user defined attribute to aware typecasted value.
```

Comment:
I think this can be rewritten as follows if that makes sense to you.  
```
Fix query attribute method on the user-defined attribute to be aware of typecasted value.
```

Changes
- `user defined` -> `user-defined` (ref: https://www.collinsdictionary.com/dictionary/english/user-defined)
- `to aware typecasted` -> `to be aware of typecasted`

Reply:
Thank you for the review! I changed that. 😄